segbot_sensors/CMakeLists.txt

```diff
@@ -82,3 +86,9 @@ if (CATKIN_ENABLE_TESTING)
   roslaunch_add_file_check(launch/kinect)
   roslaunch_add_file_check(launch/ptgrey)
 endif()
+
+add_executable(batteryDiagnostics src/segbot_sensors/battery_monitor.cpp)
+target_link_libraries(batteryDiagnostics ${catkin_LIBRARIES})
+
+add_executable(voltagePub src/segbot_sensors/fakeVoltPub.cpp)
+target_link_libraries(voltagePub ${catkin_LIBRARIES})
```

Comment:
Re adding new targets: the new targets are the binaries? I tried this and compilation fails with "install TARGETS given no RUNTIME DESTINATION for executable target "